lib/node_modules/@stdlib/math/base/special/asec/test/test.js

```diff
@@ -0,0 +1,267 @@
+
+	for ( i = 0; i < x.length; i++ ) {
+		y = asec( x[i] );
+		if ( y === expected[ i ] ) {
+			t.equal( y, expected[ i ], 'x: '+x[i]+'. E: '+expected[i] );
+		} else {
+			delta = abs( y - expected[i] );
+			tol = 1.0 * EPS * abs( expected[i] );
+			t.ok( delta <= tol, 'within tolerance. x: '+x[i]+'. y: '+y+'. E: '+expected[i]+'. tol: '+tol+'. Δ: '+delta+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function computes the inverse (arc) secant on the interval `[-1000.0,-100.0]`', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var y;
+	var i;
+
+	x = largerNegative.x;
+	expected = largerNegative.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		y = asec( x[i] );
+		if ( y === expected[ i ] ) {
+			t.equal( y, expected[ i ], 'x: '+x[i]+'. E: '+expected[i] );
+		} else {
+			delta = abs( y - expected[i] );
+			tol = 1.0 * EPS * abs( expected[i] );
+			t.ok( delta <= tol, 'within tolerance. x: '+x[i]+'. y: '+y+'. E: '+expected[i]+'. tol: '+tol+'. Δ: '+delta+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function computes the inverse (arc) secant on the interval `[-1e200,-1e208]`', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var y;
+	var i;
+
+	x = hugeNegative.x;
+	expected = hugeNegative.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		y = asec( x[i] );
+		if ( y === expected[ i ] ) {
+			t.equal( y, expected[ i ], 'x: '+x[i]+'. E: '+expected[i] );
+		} else {
+			delta = abs( y - expected[i] );
+			tol = 1.0 * EPS * abs( expected[i] );
+			t.ok( delta <= tol, 'within tolerance. x: '+x[i]+'. y: '+y+'. E: '+expected[i]+'. tol: '+tol+'. Δ: '+delta+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function computes the inverse (arc) secant on the interval `[1e300,1e308]`', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var y;
+	var i;
+
+	x = hugePositive.x;
+	expected = hugePositive.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		y = asec( x[i] );
+		if ( y === expected[ i ] ) {
+			t.equal( y, expected[ i ], 'x: '+x[i]+'. E: '+expected[i] );
+		} else {
+			delta = abs( y - expected[i] );
+			tol = 1.0 * EPS * abs( expected[i] );
+			t.ok( delta <= tol, 'within tolerance. x: '+x[i]+'. y: '+y+'. E: '+expected[i]+'. tol: '+tol+'. Δ: '+delta+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function returns `NaN` if provided `NaN`', function test( t ) {
+	var v = asec( NaN );
+	t.equal( isnan( v ), true, 'returns NaN' );
+	t.end();
+});
+
+tape( 'the function returns `NaN` if provided `+0`', function test( t ) {
+	var v = asec( 0.0 );
+	t.equal( isnan(v), true, 'returns NaN' );
+	t.end();
+});
+
+tape( 'the function returns `NaN` if provided `-0`', function test( t ) {
+	var v = asec( -0.0 );
+	t.equal( isnan( v ), true, 'returns NaN' );
+	t.end();
+});
+
+tape( 'the function returns `NaN` on the interval `(-1, 1)`', function test( t ) {
+	var v = asec( 0.5 );
+	var w = asec( -0.5 );
+
+	t.equal( isnan(v), true, 'returns NaN' );
+	t.equal( isnan(w), true, 'returns NaN' );
```

Comment:
```suggestion
	t.equal( isnan( v ), true, 'returns NaN' );
	t.equal( isnan( w ), true, 'returns NaN' );
```

To be consistent with above.

Typically, we include spaces around arguments, with exceptions for general readability when there are many nested parentheses.